source/CommonJobs/CommonJobs.MVC.UI/Content/Evaluations/evaluations.css

```diff
@@ -0,0 +1,693 @@
+/**
```

Comment:
This file was generated by SASS and the app use LESS. So, I think that we should use this file now and rewrite for LESS in the future.
What do you think?